Update product-management-team-handbook.md

justinegeffen opened the pull request:
Style, formatting, grammar, links, and general review. First commit; WIP.

justinegeffen committed:
Update product-management-team-handbook.md

Style, formatting, grammar, links, and general review. First commit; WIP.

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
+* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings
+  * Ask questions to understand development, design, QA processes
+* Meet with Amy \(Release Manager\) and read the release process doc
+* Meet with Ian \([https://it33.github.io/readme/](https://it33.github.io/readme/)\). This will be scheduled for you
```
This link is a 404. Confirm correct page is https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours.

lfbrock replied:
it should be - https://github.com/it33/readme/blob/master/README.md for Ian's readme file, and I *think* the "CEO Informational Meeting" linked in the readme is the first meeting people have (you can confirm with Amy Nicol if that's the right meeting).

justinegeffen replied:
Thanks, @lfbrock. 

@amynicol1985, in the context of meeting with Ian for the first time is the link provided aimed at a specific meeting outline or just as an overview of Ian ahead of time?

justinegeffen replied:
Will follow this up post-merge as it's not a blocker.

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
-      * Join support standups for a week or two, and help out on support tickets 
+* Learn about our customers and users
+  * Support
+    * Customers: Zendesk \(Paul\)
```
Check whether there's a new DRI for this.

lfbrock replied:
@justinegeffen it's Sven now

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
-    * How does Sales team position the product 
-    * What are typical challenges in sales 
-    * Which sales do we win, which do we lose 
+  * Sales \(Sales - Steve, Lance, Phillip and Solution Architects - Mark, John, Christian\)
```
Confirm correct department/roles are listed.

lfbrock replied:
@justinegeffen maybe we can remove the names for now and people can check in Bamboo

lfbrock replied:
Actually looking below -- Maybe using titles is better. We can saw "Regional VPs and Customer Engineers"

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
-    * What are the marketing team’s priorities 
-    * What is our current messaging and positioning around Mattermost 
-    * What is needed from product managers
+  * Marketing \(Marketing team lead\)
```
Confirm whether this is "Marketing team lead" or "Kendall - Marketing Operations Manager".

lfbrock replied:
Let's leave as generic titles, I like that better as it's easier to keep up to date. We can point them to the right people when we onboard them.

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
-    * End User & Apps \(Eric Sethna\)
-    * Mobile \(Adam Clarkson\) 
-    * Integrations \(Aaron Rothschild\) 
+  * Product areas: Functional Overview and Roadmap
```
Confirm areas are correct.

lfbrock replied:
We can use the product pillars outlined by Chandar: 
Cloud - Adam Clarkson
End user experience (Desktop, Web, and Mobile client applications) - Eric Sethna
Enterprise-grade (Platform, Admin Console) - Dennis Kittrell
Product-centric GTM (Customer Portal, Customer Journey) - Eric Sadur
App Marketplace & Developer Toolkit - Aaron Rothschild 
Workflows - Ian Tao 
Community - Jason Blais

justinegeffen committed:
Update product-management-team-handbook.md

Fixed some capitalization errors.

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -460,47 +460,48 @@ Review Release PRs and Community PRs to ensure the feature meets requirements fo
 * Maintain product accuracy on mattermost.com/product and roadmap
-* Author Forum & blog posts
+* Author Forum and blog posts
   * Example \(link to Eric’s permissions post\)
```
Confirm which post to link.

lfbrock replied:
Not sure, maybe use these examples? 
Author [forum](https://forum.mattermost.org/t/community-design-meeting-folded-reply-threads/6729/10), [blog](https://mattermost.com/blog/mattermost-launches-mattermost-superstars-program-for-contributors/), and [release announcement](https://mattermost.com/blog/mattermost-release-v5-25/) posts 

Or @wiersgallak any thoughts on what would be good examples here?

justinegeffen replied:
I've updated with your suggestions and happy to iterate in a new PR. :)

justinegeffen committed:
Update product-management-team-handbook.md

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -378,18 +378,18 @@ Activities:
 * Facilitate prioritization decisions in sprint and release planning meetings
 * Keep [Release Plan Roadmap](https://docs.google.com/spreadsheets/d/1v-ihzzAkFmO2T5UfF1wWYMgS3W6MwiNqfRQG79X3rSY/edit?usp=sharing) up to date with features queued for the release
-  * Prepare updates for the 
+  * Prepare updates for the
```
Missing a link/sentence.

lfbrock replied:
Not sure what this is, maybe we can remove it if Katie doesn't know either

justinegeffen replied:
Removed for now.

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -39,12 +39,12 @@ Product Managers are responsible for carrying out the strategic priorities of th
   * Pragmatic Marketing: “Your opinion, although interesting, is irrelevant.”
     * It’s all about the customer/user/buyer’s opinion/want/need
-* FAQ \([https://docs.mattermost.com/process/pm-faq.html](https://docs.mattermost.com/process/pm-faq.html)\) 
+* FAQ \([https://docs.mattermost.com/process/pm-faq.html](https://docs.mattermost.com/process/pm-faq.html)\)
```
Could change the link to: https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq.

justinegeffen replied:
```suggestion
* FAQ \([https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)\)
```

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
   * Update docs with answers as appropriate
-* Spend a couple weeks joining as many customer and prospect calls as possible 
+* Spend a couple weeks joining as many customer and prospect calls as possible
   * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)
```
```suggestion
  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
```

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -378,18 +378,18 @@ Activities:
+  * Fast, Obvious, Forgiving
+* Determine feature availability: Team Edition or Enterprise Edition
   * [stewardship principles](https://docs.mattermost.com/developer/manifesto.html#stewardship-principles-for-the-mattermost-open-source-project) that we use to guide whether a feature belongs in TE
   * [TE vs EE from Ian](https://pre-release.mattermost.com/private-core/pl/m1o63kk3h7numxw3z116ft4ffw)
```
```suggestion
  * [TE vs EE from Ian](https://community-release.mattermost.com/private-core/pl/m1o63kk3h7numxw3z116ft4ffw)
```

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -398,59 +398,59 @@ Activities:
 
 #### Triage
 
 * Participate in 2x weekly Triage meetings \(may be daily towards end of release\)
```
These are currently daily meetings. However, not mandatory for all PMs to join every day.

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -460,47 +460,48 @@ Review Release PRs and Community PRs to ensure the feature meets requirements fo
 #### Documentation
 
 * * Update/create documentation for features/PRs
```
```suggestion
* Update/create documentation for features/PRs
```

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -733,14 +732,14 @@ Product Managers rotate responsibility weekly for monitoring and responding to M
+5. **Secure:** Reviewed internally for security, and included in future security reviews
+6. **Accessible:**: Accessibility has been added for user interface components
+7. **Logged:** Audit log entries have been added to allow system admins to troubleshoot issues
+8. **Compliant:**: Anything needed for compliance is tracked in logs
```
```suggestion
8. **Compliant:** Anything needed for compliance is tracked in logs
```

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -733,14 +732,14 @@ Product Managers rotate responsibility weekly for monitoring and responding to M
+3. **Measurable:** Telemetry added where appropriate
+4. **Documented:** Complete documentation is available
+5. **Secure:** Reviewed internally for security, and included in future security reviews
+6. **Accessible:**: Accessibility has been added for user interface components
```
```suggestion
6. **Accessible:** Accessibility has been added for user interface components
```

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -378,79 +378,79 @@ Activities:
 
 #### Triage
 
 * Participate in 2x weekly Triage meetings \(may be daily towards end of release\)
```
```suggestion
* Participate in daily Triage meetings \(not mandatory to join every meeting\)
```

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -378,79 +378,79 @@ Activities:
 * Capture SWAG estimates from design and developers to determine length project
 * Facilitate prioritization decisions in sprint and release planning meetings
 * Keep [Release Plan Roadmap](https://docs.google.com/spreadsheets/d/1v-ihzzAkFmO2T5UfF1wWYMgS3W6MwiNqfRQG79X3rSY/edit?usp=sharing) up to date with features queued for the release
```
We use Productboard for this now instead of google spreadsheet

lfbrock replied:
`[Release Plan](https://mattermost.productboard.com/feature-board/1097526-release-tracking-internal)`

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -459,48 +459,49 @@ Review Release PRs and Community PRs to ensure the feature meets requirements fo
 #### Customer Requests
 
-Feature and improvement requests from our Enterprise customers are logged by Sales Architects, Customer Success and Support in a [Customer Request Jira project](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&issueLimit=100).
+Feature and improvement requests from our Enterprise customers are logged by Sales Architects, Customer Success, and Support in a [Customer Request Jira project](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&issueLimit=100).
```
These are logged in Productboard now. And should be Customer Engineers instead of Sales Architects.

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -459,48 +459,49 @@ Review Release PRs and Community PRs to ensure the feature meets requirements fo
-Feature and improvement requests from our Enterprise customers are logged by Sales Architects, Customer Success and Support in a [Customer Request Jira project](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&issueLimit=100).
+Feature and improvement requests from our Enterprise customers are logged by Sales Architects, Customer Success, and Support in a [Customer Request Jira project](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&issueLimit=100).
 
 Please encourage collegues to add tickets when they are discussing requests in Mattermost channels, so that we have single source with all requests. Product Managers are responsible for reviewing Customer Requests regarding features and product improvements. New tickets may already be assigned to a PM based on area of responsibility. Please review tickets often to address newly created tickets an ...
```
This whole section below should be updated to something like: 

Please encourage colleagues to file feedback in Productboard when they are discussing requests in Mattermost channels, so that we have single source with all requests. PMs are responsible for reviewing the Productboard feedback, and assigning it to relevant feature ideas. As they are reviewing the requests, PMs may also: 
- Work with Customer Success and Sales to clarify use cases, root issues and pain points, and the priority of the request to the customer
- Propose alternate solutions and workarounds when they are available
- Provide feedback on if it is a "Won't Do" feature that we are unlikely to add to the product 
- Update the feature idea to link any implementation or design tickets
- Update the "Effort" for the feature with a SWAG from the engineering team
- Update the feature idea to have a release timeframe (3 months, 6 months, future) or release version if the feature is planned 

After adding the feedback to the relevant idea in Productboard, the PM comments and mentions the reporter so they are aware the feedback has been processed, and then marks the item as "Processed".

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -514,20 +515,20 @@ In sync with the Twitter rotation, Product Managers rotate responsibility weekly
 #### Twitter Rotation
 
-Product Managers rotate responsibility weekly for monitoring and responding to Mattermost mentions on our corporate Twitter account. The PM on rotation can be found in the header of the [Mattermost Twitter channel](https://community-daily.mattermost.com/core/channels/twitter). The rotation switches on Monday and is scheduled alphabetically by first name. The PM on rotation will be mentioned on th ...
+Product Managers rotate responsibility weekly for monitoring and responding to Mattermost mentions on our corporate Twitter account. The PM on rotation can be found in the header of the [Mattermost Twitter channel](https://community.mattermost.com/core/channels/twitter). The rotation switches on Monday and is scheduled alphabetically by first name. The PM on rotation will be mentioned on the firs ...
```
Change to 
"Product Managers rotate responsibility weekly for answering product question escalations on Twitter. Twitter is monitored by the marketing team, and they will @mention the PM on rotation in the Twitter channel when they need to escalate a question. 

The PM on rotation can be found in the header of the [Mattermost Twitter channel](https://community-daily.mattermost.com/core/channels/twitter). The rotation switches on Monday and is scheduled alphabetically by first name. The PM on rotation will be mentioned on the first day of their rotation."

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -514,20 +515,20 @@ In sync with the Twitter rotation, Product Managers rotate responsibility weekly
+Product Managers rotate responsibility weekly for monitoring and responding to Mattermost mentions on our corporate Twitter account. The PM on rotation can be found in the header of the [Mattermost Twitter channel](https://community.mattermost.com/core/channels/twitter). The rotation switches on Monday and is scheduled alphabetically by first name. The PM on rotation will be mentioned on the firs ...
 
-* Monitor Twitter channel in Mattermost Contributors Team: [https://community.mattermost.com/core/channels/twitter](https://community.mattermost.com/core/channels/twitter) 
+* Monitor Twitter channel in Mattermost Contributors Team: [https://community.mattermost.com/core/channels/twitter](https://community.mattermost.com/core/channels/twitter)
```
We can remove all these bullet points, up to and including the assign any unassigned customer requests

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -633,7 +630,9 @@ Product Managers rotate responsibility weekly for monitoring and responding to M
-</table>\#\#\#\# Monthly Roadmap Q&A - Pre-sales and Support \| \| \| \| :--- \| :--- \|
+</table>
+
+### Monthly Roadmap Q&A - Pre-sales and Support
```
This meeting is now: 
- Monthly Product:Sales Call
- 2nd Monday of the month, 9am to 10am PST 
- Attendees: All PMs; Optional: Sales, BizOps, Marketing departments 
- Format: 
  - PMs share roadmap highlights
    - Focus is on features with clear release timelines, and a clear benefit to customers, so Sales knows what is coming up soon they can talk to customers about 
    -  Demos, PRFAQs, and designs are shared as needed to illustrate the benefits
  - Open Q&A

justinegeffen committed:
Update product-management-team-handbook.md

Changes as per Lindsay. Still to do: table formatting.

justinegeffen committed:
Update product-management-team-handbook.md

Removed incomplete sentence.

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
 
 Activities:
 
 * Meet with Lindsay
```
Can change to meet w/ manager

justinegeffen replied:
```suggestion
* Meet with Manager
```

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
   * Feature requests
-    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)  
+    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)
     * [Customer request triage](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97) 
```
We should replace this link with "Process feedback in productboard" with a link to productboard inbox: https://mattermost.productboard.com/insights/shared-inbox

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
+    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)
     * [Customer request triage](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97) 
-      * Track down estimates and early dev feasibility discussions on[ Customer Requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&selectedIssue=CR-132)
+      * Track down estimates and early dev feasibility discussions on [Customer Requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&selectedIssue=CR-132)
```
remove this link, we use productboard now

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
-  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)
-* Learn about the product 
+* Spend a couple weeks joining as many customer and prospect calls as possible
+  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
```
Notes should be filed in Salesforce now, Emily Hollinger has the link to the new process (Salesforce auto-posts to the channel)

justinegeffen replied:
@emilyhollinger, would you be open to sharing the link to the process, or details that would work in this context?

wiersgallak replied:
@justinegeffen here's the link: https://handbook.mattermost.com/operations/customer-success#logging-a-customer-feedback-in-salesforce

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
+* Spend a couple weeks joining as many customer and prospect calls as possible
+  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
+* Learn about the product
   * Product Direction \(Lindsay, Product Lead\)
```
Can change to manager here as well

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
+    * Workflows \(Ian Tao\)
+    * Community \(Jason Blais\)
+    * Reference:
       * [Roadmap - All teams](https://docs.google.com/spreadsheets/u/1/d/1v-ihzzAkFmO2T5UfF1wWYMgS3W6MwiNqfRQG79X3rSY/edit#gid=0)
```
```
- [Roadmap - All teams](https://mattermost.productboard.com/portal/19-combined-highlighted-features)
- [Release Plan](https://mattermost.productboard.com/feature-board/1749949-release-summary-internal)
```

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
+* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings
+  * Ask questions to understand development, design, QA processes
+* Meet with Amy \(Release Manager\) and read the release process doc
+* Meet with Ian \([https://it33.github.io/readme/](https://it33.github.io/readme/)\). This will be scheduled for you
```
```suggestion
* Meet with Ian \([https://github.com/it33/readme/blob/master/README.md](https://github.com/it33/readme/blob/master/README.md/)\). This will be scheduled for you
```

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -459,48 +458,48 @@ Review Release PRs and Community PRs to ensure the feature meets requirements fo
-  * Add/update jira tickets to capture feature requests
+* Assist with Customer Support - Tier 2 on call weekly rotation
+* PMs should be on sales/customer calls often; if not on at least a couple per week then request to be added in the CSM:PM channel and Pre-sales channel
+  * Document and post feedback via the [Customer Feedback channel](https://community.mattermost.com/private-core/channels/customer-feedback)
```
Update link here as well to be filing in Salesforce w/ Emily's new process

lfbrock commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -459,48 +458,48 @@ Review Release PRs and Community PRs to ensure the feature meets requirements fo
 #### Customer Request Rotation
 
-In sync with the Twitter rotation, Product Managers rotate responsibility weekly for monitoring newly filed [customer requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97&issueLimit=100) and ensuring newly filed requests are assigned to the appropriate PM according to areas of ownership. The weekly PM meeting is the tim ...
+In sync with the Twitter rotation, Product Managers rotate responsibility weekly for monitoring newly-filed [customer requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97&issueLimit=100) and ensuring newly filed requests are assigned to the appropriate PM according to areas of ownership. The weekly PM meeting is the tim ...
```
we can remove this section, each PM now processes requests in Productboard so we no longer have the customer request rotation

justinegeffen replied:
Removed, thanks. :)

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

justinegeffen committed:
Update product-management-team-handbook.md

Incorporated feedback from Lindsay including updated URLs for release plan, OKRs etc.

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -39,12 +39,12 @@ Product Managers are responsible for carrying out the strategic priorities of th
   * Pragmatic Marketing: “Your opinion, although interesting, is irrelevant.”
     * It’s all about the customer/user/buyer’s opinion/want/need
-* FAQ \([https://docs.mattermost.com/process/pm-faq.html](https://docs.mattermost.com/process/pm-faq.html)\) 
+* FAQ \([https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)\)
```
```suggestion
* FAQ \(https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)\
```

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,252 @@ Before diving too deeply into docs, processes, and everything else - take this c
-  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)
-* Learn about the product 
+* Spend a couple weeks joining as many customer and prospect calls as possible
+  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
```
```suggestion
  * Take detailed notes and post to [https://community.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
```

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,252 @@ Before diving too deeply into docs, processes, and everything else - take this c
         * [Release Testing assignment example](https://docs.google.com/spreadsheets/d/1gW7i8uF3jd5wuHd5mJ7UdBQsMFVZwHKMXE7WOtlDa0k/edit#gid=1130032476)
-    * Release process \(Amy, Release Manager\) 
+    * Release process \(Amy, Release Manager\)
       * [Release process](https://docs.mattermost.com/guides/core.html#release-process)
```
```suggestion
      * [Release process](https://handbook.mattermost.com/operations/research-and-development/product/release-process)
```

wiersgallak commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,252 @@ Before diving too deeply into docs, processes, and everything else - take this c
+      * Join support standups for a week or two, and help out on support tickets
     * Community: GitHub, Forums \(Amy\)
-  * Customer success \(Meet with a CSM - Emily Hollinger, Stu Doherty, Rosie Lane\) 
+  * Customer success \(Meet with a CSM - Emily Hollinger, Stu Doherty, Rosie Lane\)
```
```suggestion
  * Customer success \(Meet with a CSM - Jennifer Lawler, Jeff Johnson, Sasa Cosic, Rosie Lane\)
```

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Katie Wiersgalla <39744472+wiersgallak@users.noreply.github.com>

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -114,253 +114,252 @@ Before diving too deeply into docs, processes, and everything else - take this c
-  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)
-* Learn about the product 
+* Spend a couple weeks joining as many customer and prospect calls as possible
+  * Take detailed notes and post to [https://community.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
```
```suggestion
  * Take detailed notes and log in [Salesforce](https://handbook.mattermost.com/operations/customer-success#logging-a-customer-feedback-in-salesforce)
```

justinegeffen commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -459,48 +457,44 @@ Review Release PRs and Community PRs to ensure the feature meets requirements fo
-  * Add/update jira tickets to capture feature requests
+* Assist with Customer Support - Tier 2 on call weekly rotation
+* PMs should be on sales/customer calls often; if not on at least a couple per week then request to be added in the CSM:PM channel and Pre-sales channel
+  * Document and post feedback via the [Customer Feedback channel](https://community.mattermost.com/private-core/channels/customer-feedback)
```
```suggestion
  * Document and post feedback in [Salesforce](https://handbook.mattermost.com/operations/customer-success#logging-a-customer-feedback-in-salesforce)
```

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

amyblais commented on operations/research-and-development/product/product-management-team-handbook.md:
```diff
@@ -39,12 +39,12 @@ Product Managers are responsible for carrying out the strategic priorities of th
   * Pragmatic Marketing: “Your opinion, although interesting, is irrelevant.”
     * It’s all about the customer/user/buyer’s opinion/want/need
-* FAQ \([https://docs.mattermost.com/process/pm-faq.html](https://docs.mattermost.com/process/pm-faq.html)\) 
+* FAQ \(https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)\
```
```suggestion
* FAQ \(https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#frequently-asked-questions-faq)
```

justinegeffen committed:
Update operations/research-and-development/product/product-management-team-handbook.md

Co-authored-by: Amy Blais <amy_blais@hotmail.com>

justinegeffen commented:
@lfbrock, @wiersgallak; is this good to go? :)

wiersgallak reviewed (approved):
Thanks @justinegeffen
